tests/plugins-tester/src/merge-images-utils.js

```diff
@@ -0,0 +1,62 @@
+import { createRequire } from 'module';
+import fs from 'fs';
+
+var require = createRequire( import.meta.url );
+const mergeImages = require( 'merge-images' );
+const { Canvas, Image, loadImage } = require( 'canvas' );
+
+export const mergeFromReportDir = ( dir ) => {
```

Comment:
Is there a better name for this? it's not that clear

Reply:
How it's now? `mergeReportFromDir`